content/docs/dispensary/prescriptions.md

```diff
@@ -443,3 +443,25 @@ Click on the `Show / hide columns` button which is at the top right of the table
 ![Hide columns](/docs/dispensary/images/prescription_columns_step_3.png)
 
 If some columns are hidden, you can click `Show all` to show all of the columns again.
+
+## Using Insurance During Payment
```

Comment:
I think this section should just be called `Payment`
And we can add insurance as a section underneath.

let me know i fyou want me to just do this for you?

Reply:
Applied my understanding of the review. Please see changes here: https://github.com/msupply-foundation/msupply_docs/pull/154/commits/1ac2576957bfd7bb0637ddef304accb677bdade7